docs/src/examples.md

```diff
@@ -41,7 +41,59 @@ julia> round.(predict(ols), digits=5)
  6.83333
 ```
 
-.
+## Categorical variables/dummy coding
+
+Categorical variables will be dummy coded if they are marked as categorical in an input DataFrame. Alternatively, you can pass an explicit dictionary of contrasts.
+
+Using `categorical`:
```

Comment:
Can you make sentences, and point to the documentation for CategoricalArrays? Same below about `contrasts`, which should point to StatsModels.